### PR TITLE
Fix the README and example

### PR DESCRIPTION
**Pull request by kerwitz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,39 +9,43 @@ At midday, no matter if it is a weekend or a holiday, a tip will be posted and t
...
 
-arr.unshift(0);
-[0].concat(arr); // 98% faster in Chrome 47.0.2526.106 on Mac OS X 10.11.1
+items.unshift(0);
+[0].concat(items); // 98% faster in Chrome 47.0.2526.106 on Mac OS X 10.11.1
 ```
-Here a little bit detail, unshift edit the original array, concat return a new array. [jsperf](http://jsperf.com/unshift-item-inside-an-array)
-
-Add items at the middle of an array is easy with splice and is the most performant way to do it.
+Instead of using unshift we can also create a fresh array and join it with `arr`. [`Array.prototype.concat()`](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Array/push) will combine both into a new array and return the result.
```
Comment (loverajoel) [inline README.md:35]: change `arr` by `items`

File: README.md [hunk 2]
```diff
@@ -9,39 +9,43 @@ At midday, no matter if it is a weekend or a holiday, a tip will be posted and t
...
-Here a little bit detail, unshift edit the original array, concat return a new array. [jsperf](http://jsperf.com/unshift-item-inside-an-array)
-
-Add items at the middle of an array is easy with splice and is the most performant way to do it.
+Instead of using unshift we can also create a fresh array and join it with `arr`. [`Array.prototype.concat()`](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Array/push) will combine both into a new array and return the result. 
+Check the [jsperf](http://jsperf.com/unshift-item-inside-an-array)!
 
+For inserting elements inbetween the existing ones [`Array.prototype.splice()`](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Array/splice) turns out to be the best option.
 ```javascript
 var items = ['one', 'two', 'three', 'four'];
 items.splice(2, 0, 'hello');
```
Comment (loverajoel) [inline README.md]: update this line `items.splice(2, 0, 'hello');` by `items.splice(items.length, 0, 'hello');`

**Comment by loverajoel** [comment]
Hey @kerwitz thanks for your help with my bad english!
Make a rebase and we can merge this PR.
Thank you very match!

File: README.md [hunk 3]
```diff
@@ -9,39 +9,43 @@ At midday, no matter if it is a weekend or a holiday, a tip will be posted and t
...
-
-Add items at the middle of an array is easy with splice and is the most performant way to do it.
+Instead of using unshift we can also create a fresh array and join it with `arr`. [`Array.prototype.concat()`](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Array/push) will combine both into a new array and return the result. 
+Check the [jsperf](http://jsperf.com/unshift-item-inside-an-array)!
 
+For inserting elements inbetween the existing ones [`Array.prototype.splice()`](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Array/splice) turns out to be the best option.
 ```javascript
 var items = ['one', 'two', 'three', 'four'];
-items.splice(2, 0, 'hello');
+items.splice(items.length, 0, 'hello');
```
Comment (mastilver) [inline README.md:41]: we want to insert `'hello'` at position `2` not at the end

**Comment by loverajoel** [comment]
closed by inactivity

**Pull request closed without merging**